modules/jpa-envers/src/main/java/io/oasp/module/jpa/dataaccess/api/AdvancedRevisionEntity.java

```diff
@@ -44,6 +44,11 @@
   private transient Date date;
 
   /** @see #getUser() */
+
+  /*
+   * Uncomment the following Column annotation if the database used is Oracle 11g
+   */
+  // @Column(name = "\"user\"")
```

Comment:
I think that we have to find another solution for introducing the `@Column` annotation.
The problem I see here is that the _OASP jpa-envers_ project will be included as a dependency, so the compiled JAR will be fetched from Maven Central. Hence, the programmer will not be able to change this code in his project as suggested by the comment.

Of course, the programmer could clone the project from github, but that would come with additional effort which is not our goal.

One possibility would be to use JPA and Hibernate annotations only on `setters`. Then one could provide subclasses of `AdvancedRevisionEntity` for each database technology (oracle, postgres,...) where the according annotations would be added by overriding the setters. However, I'm not sure whether this would work, so some investigation is necessary.